src/main/java/fr/silverpricing/api/model/Chambre.java

```diff
@@ -1,8 +1,12 @@
 package fr.silverpricing.api.model;
```

Comment:
:warning: The getAllDepartements method does not handle exceptions properly and returns null in case of an error. (Severity: Major)